packages/node/test/index.test.ts

```diff
@@ -226,8 +226,10 @@ describe('#metrics', function () {
   });
 
   describe('#timeout', function () {
+    // eslint-disable-next-line mocha/no-pending-tests, mocha/no-skipped-tests
     it.skip('should silently fail metrics requests if they take longer than the timeout');
 
+    // eslint-disable-next-line mocha/no-pending-tests, mocha/no-skipped-tests
```

Comment:
ESLint rules that don't get flagged properly keep me up at night 😵‍💫

Reply:
I mostly silenced these because I have no plans to flesh these tests out and was annoyed by the ESLint rules getting flagged by Github in the PR review screeen.

Reply:
We do but these are just warnings.